Lib/test/test_dis.py

```diff
@@ -360,6 +367,17 @@ def test_boundaries(self):
         self.assertEqual(dis.opmap["EXTENDED_ARG"], dis.EXTENDED_ARG)
         self.assertEqual(dis.opmap["STORE_NAME"], dis.HAVE_ARGUMENT)
 
+    def test_widths(self):
+        for opcode, opname in enumerate(dis.opname):
+            if opname in ('BUILD_MAP_UNPACK_WITH_CALL',
+                          'BUILD_TUPLE_UNPACK_WITH_CALL'):
```

Comment:
I think it is better to misalign columns than output corrupted opcode names.

It may be worth to rename these two opcodes, but this is different issue.